dbcon/joblist/groupconcat.cpp

```diff
@@ -467,7 +470,9 @@ void GroupConcator::initialize(const rowgroup::SP_GroupConcat& gcc)
   fRm = gcc->fRm;
   fSessionMemLimit = gcc->fSessionMemLimit;
 
-  for (const auto& str : views::keys(gcc->fConstCols))
+  auto keys = gcc->fConstCols | boost::adaptors::transformed([](const auto& p) { return p.first; });
+
+  for (const auto& str : keys)
   {
     fConstantLen += str.length();
```

Comment:
I used `ranges` here and below because it looks a bit more readable than using `.first`/etc. But `boost::transform` here has the opposite effect, making the code harder to understand, IMO.

```suggestion
  for (const auto& val : gcc->fConstCols)
  {
    fConstantLen += val.first.length();
```

Reply:
++